napi.h

```diff
@@ -787,8 +796,8 @@ namespace Napi {
     ArrayBuffer();                               ///< Creates a new _empty_ ArrayBuffer instance.
     ArrayBuffer(napi_env env, napi_value value); ///< Wraps a N-API value primitive.
 
-    void* Data();        ///< Gets a pointer to the data buffer.
-    size_t ByteLength(); ///< Gets the length of the array buffer in bytes.
+    void* Data() const;        ///< Gets a pointer to the data buffer.
+    size_t ByteLength() const; ///< Gets the length of the array buffer in bytes.
```

Comment:
These changes are unrelated, though relevant. Can you please submit them in a separate PR?